manticore/core/executor.py

```diff
@@ -304,7 +304,7 @@ def is_shutdown(self):
     ###############################################
     # Priority queue
     @sync
-    def _put(self, state_id):
+    def put(self, state_id):
```

Comment:
So we removed the underscore. Does it means the API is public and can be used from the outside?